lib/token.js

```diff
@@ -197,9 +227,15 @@ function handleOAuthResponse(sdk, oauthParams, res, urls) {
       return tokenDict[tokenTypes];
     }
 
-    if (!tokenDict['token'] && !tokenDict['id_token']) {
-      throw new AuthSdkError('Unable to parse OAuth flow response');
-    }
+    // Validate response against tokenTypes
+    var validateTypes =  ['token', 'id_token'];
```

Comment:
nit: Change `validateTypes` to `valiteTokenTypes` for verboseness.

Reply:
I'd suggest `validateTokenTypes` over `valiteTokenTypes` :)

Reply:
have updated